utils/sv_IO_panel_tools.py

```diff
@@ -218,17 +219,17 @@ def create_dict_of_tree(ng, skip_set={}, selected=False):
         for k, v in node.items():
 
             if not isinstance(v, (float, int, str)):
-                print('//')
-                print(node.name, ' -> property:', k, type(v))
+                debug('//')
```

Comment:
i'm a little torn on this..i suppose if I really need to see this message i'd switch on debug mode..

Reply:
(updated my comment)